diffrax/_adjoint.py

```diff
@@ -852,3 +852,38 @@ def loop(
         )
         final_state = _only_transpose_ys(final_state)
         return final_state, aux_stats
+
+
+class ForwardMode(AbstractAdjoint):
+    """Differentiate through a differential equation solve during the forward pass.
+    (So it is not really an adjoint - it is a different way of quantifying the
```

Comment:
I might specify what an adjoint is here, to make this clear.

Reply:
Gave it a shot!